src/main/java/levy/gameoflife/GameOfLifeFrame.java

```diff
@@ -0,0 +1,26 @@
+package levy.gameoflife;
+
+import javax.swing.JFrame;
+import java.awt.BorderLayout;
+
+public class GameOfLifeFrame extends JFrame {
+
+    private final GameOfLife game = new GameOfLife(50, 40);
+
+    public GameOfLifeFrame() {
+        setSize(800, 600);
+        setTitle("Game of Life");
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+        int[][] initialGrid = new int[40][50];
```

Comment:
You are already creating the 50x40 grid in GameOfLife, why are you creating it again?